release.sh

```diff
@@ -47,7 +47,8 @@ sbt '+publishSigned'
 
 echo "Publishing docs..."
 sbt makeSite
-mv documentation/target/sphinx/html ../releasing-the-docs
+rm -rf documentation/target/paradox/paradox # Paradox site bug?
```

Comment:
This is a workaround for what might be a bug in the sbt-site Paradox plugin. If I have time I will try to investigate later.